### PR TITLE
Add non-compensation issue categories

### PR DESCRIPTION
**Pull request by leikkisa**
connects #5882

### CODE REVIEW

File: app/services/legacy_optin_manager.rb [hunk 1]
```diff
@@ -26,7 +26,7 @@ def process!
           next unless vacols_id == legacy_appeal.vacols_id
 
           # gut checks
-          unless open_legacy_issues.map(&:vacols_sequence_id).map(&:to_s).include?(vacols_sequence_id)
+          unless open_legacy_issues.map(&:vacols_sequence_id).map(&:to_s).include?(vacols_sequence_id.to_s)
```
Comment (leikkisa) [inline app/services/legacy_optin_manager.rb]: Temporary fix because of a failing test, this will be obsolete once the legacy issue opt in PR is merged.

**Comment by leikkisa** [comment]
I made an assumption that the existing issue categories are for "Compensation", and that those are the ones to still use for Appeals because I didn't see "Compensation" in the list.

I am also wondering if I should add "Unknown issue category" at the top of every list? I kind of remember that coming up in conversation, but I couldn't remember if it was that or "Other" which is in some of the lists.

File: spec/support/intake_helpers.rb [hunk 2]
```diff
@@ -1,5 +1,108 @@
 # rubocop:disable Metrics/ModuleLength
 module IntakeHelpers
+  # rubocop: disable Metrics/MethodLength
```
Comment (pkarman) [inline spec/support/intake_helpers.rb]: I like to see these helpers consolidated.

File: spec/feature/intake/nonrating_request_issue_modal_spec.rb [hunk 3]
```diff
@@ -0,0 +1,91 @@
+require "rails_helper"
```
Comment (pkarman) [inline spec/feature/intake/nonrating_request_issue_modal_spec.rb]: I like this!

**Pull request merged**